### ISSUE TITLE
[Bug] Firefox (any version) sometimes doesn't apply dark reader

### ISSUE THREAD

**Issue by JosefLitos** [darkreader/darkreader#7340]
# Bug Report

## Bug Description

Since the last week, I have started to ocassionaly experience this extension not applying randomly. To be exact, Whenever a tab is loaded/reloaded/new site opened, there appears to be a chance of the extension doing nothing, despite showing as applied in the menu. When I refresh the page or disable and reenable it for the site - simply put, when I make an update - it applies as expected.

## Website Address

Not website specific, is based on any site load.
## Steps To Reproduce

Load a site in, if you give it enough tries, it may occur. As I have not installed any new addons or changed anything since the last time it worked without exception, I don't believe it was caused by localCDN or uBlock Origin.
## Expected Behavior

Activate the extension on every load of any page not restricted by the set rules. Without any occasional odds left out.
## Actual Behavior

When any site is loaded, there is a chance of the extension doing no work - meaning not inverting the page despite showing in the menu as applied.
## Screenshots

![obrazek](https://user-images.githubusercontent.com/54900518/141322846-4630d661-e1a5-4033-b5a3-8d64e7920e31.png)
## System Information

- OS: ArchLinux kernel: 5.10.78-1-lts
- Browser: Firefox 88, Firefox 93, Firefox 94
- Dark Reader Version: 4.9.42

**Comment by Gusted** [comment]
Hmm that's weird issue - especially if it's not reproduce-able.

The next when it happens, could you check if there are any errors in the console?
And as well if you go to `about:debugging#/runtime/this-firefox`, click on "Inspect" by the dark reader card. Goes to Console and check if their are any _recent_ errors.

Regards,
Gusted

**Comment by JosefLitos** [comment]
![obrazek](https://user-images.githubusercontent.com/54900518/141372094-d9679cc2-ff91-4347-a311-344b1e5aab9e.png)
this was something that was there already, but when I see it happen again, I shall make an update.

**Comment by JosefLitos** [comment]
This is all I got from the console when I just loaded a page on a tab and the addon didn't apply.
![obrazek](https://user-images.githubusercontent.com/54900518/141509253-1d5c65f9-3961-4977-92cf-3ea2f4ed14ff.png)

**Comment by Gusted** [comment]
> this was something that was there already, but when I see it happen again, I shall make an update.

Hmm, that's a "general" error and don't seem to resemble any dark reader error.

**Comment by JosefLitos** [comment]
This is what I got just now, once again, when starting firefox and loading the first page. The only error, I got, was this one:
![obrazek](https://user-images.githubusercontent.com/54900518/141736525-1339e479-9116-4d5a-aee7-83965c447222.png)
I hope this helps

**Comment by Gusted** [comment]
Not really, such errors just happens but aren't a reason for firefox to make dark reader stop working.

**Comment by pras92** [comment]
OS: Ubuntu 20.04.1 LTS
Firefox: 94.0 (x64)
Dark Reader: 4.9.42

Dark Reader works in Brave (Ubuntu + Win10), FF(Win10), but not in FF Ubuntu.
Console output:

  /''''\
 (0)==(0)
/__||||__\
Welcome to Dark Reader! index.js:4278:13
Error: Please use $(ref:runtime.getURL). background.js:11
    <anonymous> moz-extension://1322c379-8187-4907-884e-c1af600ad2f9/background.js:11
The Web Console logging API (console.log, console.info, console.warn, console.error) has been disabled by a script on this page.
Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content. index.html
Promise rejected after context unloaded: Actor 'Conduits' destroyed before query 'RuntimeMessage' was resolved
index.js:1312
Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content. 2 popup-fenix.html

**Comment by JosefLitos** [comment]
Until I refreshed (or loaded a new) page, changing from `auto` (currently I have light mode`) to `enabled` or to `disabled` and then to `enabled` nothing changed. After I refreshed a page, dark mode started working on all sites and switching the mode worked as expected. I had the settings described in the PR enabled already.
